daemon/main.go

```diff
@@ -341,6 +341,7 @@ func init() {
 	flags.BoolVar(&enableTracing,
 		"enable-tracing", false, "Enable tracing while determining policy (debugging)")
 	flags.String("envoy-log", "", "Path to a separate Envoy log file, if any")
+	flags.String("403-msg", "", "Message returned in 403 body")
```

Comment:
I'd use a more explicit name. This is HTTP specific.
So I'd suggest for example `--http-403-msg`.

Reply:
I second rlenglet. Could you also explain in the help text that this is for L7 HTTP proxying (as opposed to the agent API).

Reply:
OK.